WordPress/src/main/java/org/wordpress/android/ui/uploads/MediaUploadHandler.java

```diff
@@ -141,8 +146,22 @@ static boolean hasPendingOrInProgressMediaUploadsForPost(PostModel postModel) {
      * it's assumed to be a completed upload (returned progress is 1).
      */
     static float getProgressForMedia(MediaModel media) {
-        Float progress = sProgressByMediaId.get(media.getId());
-        return progress == null ? 0 : progress;
+        if (!sProgressByMediaId.containsKey(media.getId())) {
+            return 1;
+        }
+
+        float uploadProgress = sProgressByMediaId.get(media.getId());
```

Comment:
now that there's a distinction between optimisation progress and upload progress, let's rename this var `sProgressByMediaId` to `sUploadProgressByMediaId` to match the naming structure of `sOptimizationProgressByMediaId`